superduperdb/ext/unstructured/encoder.py

```diff
@@ -1,7 +1,11 @@
+# ruff: noqa: E402
```

Comment:
please tell an alternative to this ? @blythed  if we want to maintain noqa limit

Reply:
Let's disable that error code, since it doesn't make sense now that we put the `requires_packages` before the import statements.